test/spec/Feature/Query/SingularSpec.hs

```diff
@@ -69,7 +69,7 @@ spec =
             [("Prefer", "tx=commit"), singular]
             [json| { address: "zzz" } |]
           `shouldRespondWith`
-            [json|{"details":"Results contain 4 rows, application/vnd.pgrst.object+json requires 1 row","message":"JSON object requested, multiple (or no) rows returned","code":"PGRST116","hint":null}|]
+            [json|{"details":"Results contain 4 rows, application/vnd.pgrst.object+json requires 1 row","message":"JSON object requested, multiple (or no) rows returned","code":"PGRST116","hint":"4"}|]
```

Comment:
Hm, just noticing that we already have the affected rows on `details`. @laurenceisla Perhaps they were added after #1655 was opened and we forgot to close?

Reply:
Hmm, I understood that it was already present in the details, as shown in the issue's OP, but that an easy access to the number of rows was needed (as an alternative to the `404`, `412`), hence only the number of rows in the hint. The number of rows is present in the details even in version 7.0.0, which was launched before the issue was opened,  see [this diff from 2017](https://github.com/PostgREST/postgrest/commit/104a7ed4fa247e30cd5a3e150ebdf3d7515b1953#diff-328c2f3103b25df9c32be37811eabc32ea0f39cea4f860b33651b3d470adf974R46-R55).